main.tf

```diff
@@ -3,11 +3,19 @@ module "label" {
   namespace  = "${var.namespace}"
   name       = "${var.name}"
   stage      = "${var.stage}"
-  delimiter  = "${var.delimiter}"
```

Comment:
We should be able to override delimiter by setting `delimiter` argument.
Do not drop this.

Reply:
We don't need to define it - default value is OK for us